microsoft-edge/extensions-chromium/developer-guide/sidebar.md

```diff
@@ -37,15 +38,27 @@ Extensions can optionally use the sidebar API to show a custom UI in the Microso
 <!-- ====================================================================== -->
 ## Features of the Sidebar API
 
-* An extension in the sidebar has access to all of the [Supported APIs for Microsoft Edge extensions](./api-support.md).
+<!-- compare https://developer.chrome.com/docs/extensions/reference/sidePanel/#overview -->
+
+Features of the sidebar API include:
 
 * The sidebar remains open while navigating between tabs.
+<!-- Chrome list item 1:
+* The side panel remains open when navigating between tabs (if set to do so). -->
 
-* An extension in the sidebar can be made available for specific sites.
+* An extension in the sidebar can be made available for specific websites.
+<!-- Chrome list item 2:
+* It can be available only on specific websites. -->
 
-For details on how to create an extension, see:
-* [Create an extension tutorial, part 1](../getting-started/part1-simple-extension.md)
-* [Create an extension tutorial, part 2](../getting-started/part2-content-scripts.md)
+* An extension in the sidebar has access to all of the [Supported APIs for Microsoft Edge extensions](./api-support.md).
+<!-- Chrome list item 3:
+As an extension page, side panels have access to all Chrome APIs. -->
+
+<!-- 
+* In Settings for Microsoft Edge, users can specify which side the panel should be displayed on. -->
```

Comment:
Add this list item?
* In Settings for Microsoft Edge, users can specify which side the panel should be displayed on.

An equivalent item is in https://developer.chrome.com/docs/extensions/reference/sidePanel/#overview --
Within Chrome's settings, users can specify which side the panel should be displayed on.

@abhishek-shanthkumar, @Bhuvana-Priya

Reply:
removed